qgis_hub_plugin/gui/resource_browser.py

```diff
@@ -50,6 +51,16 @@ def __init__(self, parent=None, iface=None):
 
         self.pushButtonDownload.clicked.connect(self.download_resource)
 
+        self.checkBoxGeopackage.stateChanged.connect(
```

Comment:
If we check in the hub API, there are 5 resource types
![image](https://github.com/qgis/QGIS-Hub-Plugin/assets/1421861/6e4e4f7b-8725-424b-b281-c61774d44404)

It seems the API doesn't return all five resource types. Could you check it? Add the missing 2 check boxes after that.

Reply:
I checked the API URL used, it only returns three types of resources.
Is there any other API I can use, or can you direct me to the place where I can find the API's.

Reply:
It's not really documented, but it's from this https://github.com/qgis/QGIS-Django/blob/master/qgis-app/api/urls.py

Reply:
I will create an issue in that repo to include the other resource type also.

Reply:
https://github.com/qgis/QGIS-Django/issues/280